src/db_options.rs

```diff
@@ -3329,6 +3331,14 @@ impl Options {
         }
     }
 
+    /// Sets the blob cache.
```

Comment:
would be good to copy the information here https://github.com/facebook/rocksdb/blob/v9.3.1/include/rocksdb/advanced_options.h#L979-L987 to give more context on what this is.

Reply:
Ok, I've copied the text from rocksdb.